internal/veracode-api-wrapper/updatemitigationinfo/scripts/updatemitigationinfo.py

```diff
@@ -0,0 +1,100 @@
+BUILD_REGEX = '(build build_id=\")(\d+)'
+SUCCESS_RESPONSE_KEY = 'desc='
+PREFIX = '[VERACODE API WRAPPER] '
+ACTION_FALSEPOSITIVE = 'fp'
+ACTION_APPDESIGN = 'appdesign'
+
+print PREFIX, '- START'
+parser = argparse.ArgumentParser(description='Veracode API wrapper to update mitigation comment.')
+parser.add_argument('-a', action='store', dest='applicationID',
+                    help='Veracode Application ID')
+
+parser.add_argument('-b', action='store', dest='buildID',
+                    help='Build ID of the scan')
+
+parser.add_argument('-u', type=str, action='store', dest='username',
+                    help='Username')
+
+parser.add_argument('-cf', type=str, action='store', dest='csvFilePath',
+                    help='CSV file path of feed back report')
+
+parser.add_argument('-of', type=str, action='store', dest='outputFilePath',
+                    help='Output file path')
+
+args = parser.parse_args()
+
+password = raw_input("Enter password: ")
+
+# Get build id of particular application if build id is not provided.
+if (args.buildID == None):
+    print PREFIX, '- Retrieving the build id of application id : ', args.applicationID
+    payload = {'app_id': args.applicationID}
+    response = requests.post(GET_BUILD_API, params=payload, files=None, auth=(args.username, password))
+    matchObj = re.search(BUILD_REGEX, response.text, re.M | re.I)
+    args.buildID = matchObj.group(2)
+    print PREFIX, " Build Id of given application : " + args.buildID
+
+
+# Update mitigation comment.
+def updateComment(flowId, comment, action):
+    updateCommentPayload = {'build_id': args.buildID, 'action': action, 'comment': comment, 'flaw_id_list': flowId}
+    print updateCommentPayload
+    response = requests.post(UPDATE_MITIGATION_API, params=updateCommentPayload, files=None,
+                             auth=(args.username, password))
+    if response.status_code == '200':
+        if SUCCESS_RESPONSE_KEY not in response.text:
+            output = "SUCCESS , Response : ", response.content
+        else:
+            output = "ERROR , Response : ", response.content
+    else:
+        output = "ERROR , Response : ", response.content
+    writeFile(flowId, output)
+    pass
+
+
+def writeFile(flowId, output):
+    file = open(args.outputFilePath, "a")
+    file.write("Flow ID : " + flowId + " " + output)
+    file.write("\n")
+    file.close()
+
+
+print PREFIX, "Reading the CSV entries from analysed report"
+
+# Read CSV file and call api to update mitigation comment.
+with open(args.csvFilePath, 'r') as csvfile:
+    reader = csv.DictReader(csvfile)
+    for row in reader:
+        flowId = dict(row).get("issue_id")
+        if (dict(row).get("WSO2_resolution") == 'False Positive'):
+            action = ACTION_FALSEPOSITIVE
+        else:
+            action = ACTION_APPDESIGN
+        comment = dict(row).get("WSO2_resolution") + "\n" + dict(row).get("Use_Case") + "\n" + dict(row).get(
```

Comment:
We need to add section headers so that comment looks as follows: 

```
Resolution: Already Mitigated

Details: [the details comment on why this is already mitigated goes here]

Use case: [use case of the issue goes here]
```

There is no need of adding "Resolution" section for `false positives`. Hence comment should look as follows: 
```
Details: [the details comment on why this is already mitigated goes here]

Use case: [use case of the issue goes here]
```

There should be two `\n` to have this formatting. Also check if `Vulnerability_Influence` is the correct field to fetch.

Reply:
In WSO2 Security Feedback report, we divide the WSO2 Comment into WSO2_resolution, use case, and Vulnerability influence.